Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
I'm not entirely sure I understand how `keg_only` works, exactly. If a keg isn't linked, is it even possible for it to be a runtime dependency? How does that work, generally?